app/assets/javascripts/student_profile/Api.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+class Api extends React.Component {
```

Comment:
This shouldn't be a React component, it's not rendering UI to the DOM.

To translate what this was before, we can write each function like this:

```
export function saveNotes(...) { ... }
export function deleteEventNoteAttachment(...) { ... }
```

And then underscore-prefixed methods then don't need to be exported at all (or if you want to for testing purposes, you can export them as underscore still).

Reply:
Ah I read this wrong and thought Api was static.  This is great here, just don't need `extends React.Component` and the props bits.